src/app/common/api.ts

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
```

Comment:
Can we potentially adapt this to work with authorization as well? It does seem like a lot though to have the entire set of api requests in one file. I was thinking of creating a more general wrapper that would allow you to request and then it would automatically handle the authorization checking (putting the authorization header and logging out (i.e. delete the token and redirect to login with 401 errors)